Podfile

```diff
@@ -4,6 +4,7 @@
 target 'ios-showcase-template' do
   # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
   use_frameworks!
+  platform :ios, '10.0'
```

Comment:
Default platform is required and crucial to set base for publishing and platform support. This value should go up with every release of the IOS platform.